### PR TITLE
feat/gh actions docker build aepp server

### PR DESCRIPTION
**Pull request by mmpetarpeshev**

### CODE REVIEW

**Comment by mmpetarpeshev** [comment]
Will close this one as we moved the tls on wildcard , because of limitation in lets encrypt.
No issues for the apps.

File: server/Dockerfile [hunk 1]
```diff
@@ -9,5 +9,5 @@ COPY . /app
 RUN npm install
 
 RUN apt purge -y software-properties-common ca-certificates wget curl ssh git python build-essential && apt autoremove -y
-
+ENV OPENSSL_CONF=/dev/null
```
Comment (thepiwo) [inline server/Dockerfile:12]: why is that needed?

**Review by thepiwo — APPROVED** [review]
if it works its fine :)

File: server/Dockerfile (hunk 1 above)
Comment (mmpetarpeshev) [reply server/Dockerfile:12]: As we tested it, there is some issue without that setting for the OpenSSL conf dir. I send you the error and from what we discussed, let's merge it like that for now.

Comment (thepiwo) [reply server/Dockerfile:12]: yes fine

File: .github/workflows/aepp-prod-docker.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          release-type: node
+          package-name: ""
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"ci","section":"CI / CD","hidden":false},{"type":"test","section":"Testing","hidden":false},{"type":"refactor","section":"Refactorings","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
+
+      - name: Set up Docker Buildx
+        id: buildx
+        # Use the action from the master, as we've seen some inconsistencies with @v1
+        # Issue: https://github.com/docker/build-push-action/issues/286
+        uses: docker/setup-buildx-action@master
+        # Only worked for us with this option on �‍♂️
```
Comment (venimus) [inline .github/workflows/aepp-prod-docker.yml:40]: that char keeps appearing
